### PR TITLE
Create Ad Hoc service for assigning case to worker (WIP)

### PR DESCRIPTION
**Pull request by ClaudioCarmeli**

### CODE REVIEW

File: app/services/support/update_case.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Assign a {Support::Case} to a {Support::Agent}
+#
+module Support
+  class UpdateCase
```
Comment (peterdavidhamilton) [inline app/services/support/update_case.rb:4]: We shouldn't need to be referring to the namespace. Can you look into this please.

Comment (peterdavidhamilton) [reply app/services/support/update_case.rb:4]: @AndrewCowle

File: app/services/support/update_case.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    class UnexpectedContentfulModel < StandardError; end
```
Comment (peterdavidhamilton) [inline app/services/support/update_case.rb:5]: This error class has snuck in here without a name change

File: app/services/support/update_case.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    attr_reader :kase
+
+    # @param kase [Support::Case]
+    #
+    def initialize(kase:)
+      @kase = kase
+    end
+
+    # @see UpdateCase#call
```
Comment (peterdavidhamilton) [inline app/services/support/update_case.rb:15]: This documentation is self-referencing

**Comment by peterdavidhamilton** [comment]
Any movement on this since I saw it last @ClaudioCarmeli?

**Comment by ClaudioCarmeli** [comment]
Closing this one, in favour of https://github.com/DFE-Digital/buy-for-your-school/pull/530

**Pull request closed without merging**